festim/exports/derived_quantities/average_surface.py

```diff
@@ -3,9 +3,40 @@
 
 
 class AverageSurface(SurfaceQuantity):
+    """
+    Computes the average value of a function on a given surface
```

Comment:
You could add the equation here

```suggestion
    Computes the average value of a function on a given surface
    int( f dx) / int (1 * dx)
```